CHANGELOG.md

```diff
@@ -0,0 +1,207 @@
+# Changelog
+
+## v1.11.0-rc1
+
+Summary of Changes
+------------------
+
+**Major Changes:**
+* Enable CiliumEndpointSlice feature (#17658, @krishgobinath)
+
+**Minor Changes:**
+* Add flag to list all available configurations (#17303, @h3llix)
+* Add WireGuard status to cilium encrypt. (#17684, @h3llix)
+* crd: Add categories for cilium CRDs (#17162, @sayboras)
+* datapath,daemon: Enable multi-dev XDP (#17655, @brb)
+* datapath: Add a flag to set VXLAN and Geneve ports (#16874, @errordeveloper)
+* Detect devices from global unicast routes in addition to only
 looking for the device with the Kubernetes Node IP and the one with
 default route.  This expands the set of devices used for kube-proxy
 replacement, host firewall and bandwidth manager and should reduce
 the need to specify devices manually. (#17219, @joamaki)
+* feat: allow installing hubble ui as standalone (#17473, @eddycharly)
+* helm: Use `batch/v1` apiVersion for CronJob in K8s 1.21+ (#16635, @gandro)
+* option: Rename egress gateway flag to `enable-ipv4-egress-gateway` (#17695, @pchaigno)
+* Pod visibility annotations are now supported for Kafka and other policies implemented via Cilium Go extensions for Envoy. (#16935, @trvll)
+* Reduce bugtool memory usage (#17546, @tklauser)
+* Support advertising Pod CIDRs via BGP (#16525, @christarazi)
+* Support graceful termination for service load-balancing such that active connections don't break when endpoints are deleted. (#17716, @aditighag)
+* Support TLS certificate auto-generation using certmanager (#17238, @dungdm93)
+* vendor: Update k8s dependencies and tests to 1.22.0-rc.0 (#16989, @nathanjsweet)
+* wireguard: Add fallback to userspace implementation (#17451, @gandro)
+
+**Bugfixes:**
+* Adds an `ACCEPT` rule for untracked pkts in `filter:CILIUM_OUTPUT` (#17585, @Weil0ng)
+* bug/pkg/health: Fix Nil Address Issue in Node Update Mechanism (#17667, @nathanjsweet)
+* Define operator feature flags to allow the operator to register related CRDs. (#17772, @pchaigno)
+* egress gateway: fix non-tunnel (direct routing) mode (#17517, @kkourt)
+* egressgateway: Allow several CENPs with same egress IP (#17773, @pchaigno)
+* Envoy configuration is fixed to work also when IPv6 is disabled. (#17281, @rock-andy)
+* Fix identity leak via FQDN selectors (#17699, @joestringer)
+* fix incorrect application of egress gateway policy to internal cluster traffic.
 require  a 5.2 kernel or later for the egress gateway policy feature. (#17639, @kkourt)
```

Comment:
ditto about `^M`

Reply:
CC @kkourt  this is how newlines show up in the release notes, typically we end up having to strip them out or mess with the release notes to get them to render right. Shorter more concise release notes can mitigate this.

Reply:
Ah thanks, was not aware of this. Is there a way to add multiple release notes in the PR then? In my case what I was going for was two different notes (bullets).

Reply:
Unfortunately I don't think so, other than creating dedicated PRs for each point.

If we were guaranteed to always write messages this way then maybe we could adapt the release-notes tool to split it into two items, but in this case we actually had this item which was trying to include two points, plus a different item which just had newlines because that's the way the one sentence was split.